tests/all.js

```diff
@@ -1362,6 +1362,32 @@ test('alias chain (simple)', function() {
   });
 });
 
+test('alias chain (simple) with implicit /index', function() {
+  define('bar/index', [], function() {
+    return 'I AM BAR';
+  });
+
+  define('quz', define.alias('foo'));
+  define('foo', define.alias('bar'));
+
+  equal(require('quz'), 'I AM BAR');
+
```

Comment:
should we also test `require('qux/index')` ?

Reply:
on it

Reply:
good to 🚢  once that lands.

Reply:
I'm torn now as to whether `qux` is a typo, can you unpack this a bit?

Reply:
sorry, yes very much a typo, I meant:

`require('quz/index')`